public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -3,18 +3,18 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
 
 <div class="gf-form-group">
 	<div class="gf-form max-width-30">
-		<span class="gf-form-label width-7">Host</span>
+		<span class="gf-form-label width-10">Host</span>
```

Comment:
@dprokop I changed all the labels in this section to have width 10, to make sure they line up with each other. The labels are no longer wrapped either. What do you think?